pkg/virt-controller/watch/vmi_test.go

```diff
@@ -3576,12 +3576,10 @@ func simpleIfaceStatus(ifaceName string) *virtv1.VirtualMachineInstanceNetworkIn
 	}
 }
 
-func readyHotpluggedIfaceStatus(
```

Comment:
Regarding the `updateInterfaceStatus` tests - 
On test called "VMI with an interface on spec (matched on status) with the pod interface ready"," the VM does'nt actually has a status,. So please change its name to "VMI with an interface on spec with the pod interface ready","

Also please add a test called - "VMI with an interface on spec (matched on status) with the pod interface ready",". For a VM that already has the interface in its status (with domain and ga info source).